modules/apps/site-initializer/site-initializer-testray/extra/remote-app/src/pages/Project/Suites/SuiteOutlet.tsx

```diff
@@ -13,24 +13,61 @@
  */
 
 import {useEffect} from 'react';
-import {Outlet, useOutletContext, useParams} from 'react-router-dom';
+import {
+	Outlet,
+	useLocation,
+	useOutletContext,
+	useParams,
+} from 'react-router-dom';
 
-import {useHeader} from '../../../hooks';
 import {useFetch} from '../../../hooks/useFetch';
+import useHeader from '../../../hooks/useHeader';
 import i18n from '../../../i18n';
-import {TestraySuite} from '../../../services/rest';
+import {TestrayProject, TestraySuite} from '../../../services/rest';
+import useSuiteActions from './useSuiteActions';
 
 const SuiteOutlet = () => {
-	const {projectId, suiteId} = useParams();
-	const {testrayProject}: any = useOutletContext();
-	const {setHeading} = useHeader({timeout: 100});
+	const {actions} = useSuiteActions({isHeaderActions: true});
+	const {pathname} = useLocation();
+	const {projectId, suiteId, ...otherParams} = useParams();
```

Comment:
The only parameter needed here is: 

since the projectId that was used in basePath was removed

```suggestion
	const {suiteId} = useParams();
```